benchmarks/benchmarks/optimize_linprog.py

```diff
@@ -274,3 +282,41 @@ def setup(self, meth, prob):
 
     def time_netlib_presolve(self, meth, prob):
         _presolve(self.lp_cleaned, rr=False, tol=1e-9)
+
+
+class Netlib_infeasible(Benchmark):
+    params = [
+        methods,
+        infeasible_problems
+    ]
+    param_names = ['method', 'problems']
+
+    def setup(self, meth, prob):
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        datafile = os.path.join(dir_path, "linprog_benchmark_files",
+                                "infeasible", prob + ".npz")
+        data = np.load(datafile, allow_pickle=True)
+        self.c = data["c"]
+        self.A_eq = data["A_eq"]
+        self.A_ub = data["A_ub"]
+        self.b_ub = data["b_ub"]
+        self.b_eq = data["b_eq"]
+        self.bounds = np.squeeze(data["bounds"])
+        self.status = None
+
+    def time_netlib_infeasible(self, meth, prob):
+        method, options = meth
+        res = linprog(c=self.c,
+                      A_ub=self.A_ub,
+                      b_ub=self.b_ub,
+                      A_eq=self.A_eq,
+                      b_eq=self.b_eq,
+                      bounds=self.bounds,
+                      method=method,
+                      options=options)
+        self.status = res.status
+
+    def track_netlib_infeasible(self, meth, prob):
+        if self.status is None:
+            self.time_netlib_infeasible(meth, prob)
```

Comment:
If this code path is hit, it will just run all the problems twice it looks like. Not the end of the world, but at least worth a comment.

Reply:
Unfortunately, yes. The alternative is to perform timing in a `track_` function manually; I think that's how the DE benchmarks work, but I didn't want to do that. I am hoping for an update to ASV that allows the user to specify whether separate `Benchmark` objects need to be created for `time_` and `track_` benchmarks.